WISE-Community/WISE-Client
# Manage Students: create team and display unassigned students

## Changes
- Show unassigned students, and move students to/from it
- Added ability for teacher to create a new team in the new Manage Students page

## Test
- First, review and pull the PR [New Endpoint for creating new team and showing unassigned students](https://github.com/WISE-Community/WISE-API/pull/28)

## Test unassigned students
- Add a student to the run, but do not launch the unit
- As teacher, in the Manage Students view, the student should appear in the "Unassigned Students" section
- Move student to/from the "Unassigned Students" section should work

## Test create new team
- As teacher, go to the new Manage Students page. Create Team button should be there
- Click on the button. This should open a dialog for creating a new team. All of the students in the period should be listed here, including unassigned students
- Use the dialog to choose initial members for the new team. If the student is already in a team, a warning should appear.
 
Closes #308
